dcmdump/src/dump.rs

```diff
@@ -70,6 +70,23 @@ pub fn dump_file(obj: DefaultDicomObject, width: u32, no_text_limit: bool) -> Io
     Ok(())
 }
 
+pub fn dump_object(obj: &InMemDicomObject<StandardDataDictionary>, no_text_limit: bool) -> IoResult<()> {
```

Comment:
I think that all `pub` functions should let the user could dump something into an arbitrary writer.

```rust
pub fn dump_object(to: impl Write, obj: &InMemDicomObject<StandardDataDictionary>, no_text_limit: bool) -> IoResult<()> {
```

This should not be too far from adding this `to` parameter adn replacing `println!` with `writeln!`.